erpnext/selling/doctype/selling_settings/test_selling_settings.py

```
@@ -1,14 +1,89 @@
+			"validate_selling_price": 1,
+		})
+
+		# monkeypatch get_doc_before_save here is unnecessary but added for safety;
+		# if other tests in future override, we need to ensure this test is not affected
+		class_under_test.get_doc_before_save = lambda: None
+
+		try:
+			# Use insert instead of validate to test the whole flow,
+			# since validate is called by insert
+			class_under_test.insert()
+		except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)